drizzlepac/haputils/photometry_tools.py

```diff
@@ -218,13 +223,17 @@ def convert_flux_to_abmag(in_flux, photflam, photplam):
         input flux values converted to ABMAG
     """
 
-    # convert flux from units of electrons/second to ergs/cm2/angstrom/second
-    f_lambda = in_flux * photflam
+    # Avoid taking the log10 of zero
+    if math.isclose(photflam, 0.0, abs_tol=1e-9):
```

Comment:
Typical values of PHOTFLAM are on the order of 1e-16 to 1e-20 or less as reported in the WFC3 Photometric Zero Point page.  In which case, the `abs_tol` value is too large and would always flag any value of PHOTFLAM as `identical` to zero.

Reply:
This must be my issue.

Reply:
Addressed in 3a0707e.